README.md

```diff
@@ -186,6 +186,32 @@ getFID(sendToGoogleAnalytics);
 getLCP(sendToGoogleAnalytics);
 ```
 
+### Send the results to Google Tag Manager
+
+The following example measures each of the Core Web Vitals metrics and sends them as seperate dataLayer-events to be used by Google Tag Manager. With the `web-vitals` trigger you send the metrics to any tag inside your account.
+
+```js
+import {getCLS, getFID, getLCP} from 'web-vitals';
+
+function sendToGoogleAnalytics({name, delta, id}) {
+  // Assumes the global `dataLayer` array exists, see:
+  // https://developers.google.com/tag-manager/devguide
+  dataLayer.push({
+      event: 'web-vitals',
+      event_category: 'Web Vitals',
+      // Google Analytics metrics must be integers, so the value is rounded.
+      // For CLS the value is first multiplied by 1000 for greater precision
+      // (note: increase the multiplier for greater precision if needed).
+      event_value: Math.round(name === 'CLS' ? delta * 1000 : delta),
+      // The `id` value will be unique to the current page load. When sending
+      // multiple values from the same page (e.g. for CLS), Google Analytics can
+      // compute a total by grouping on this ID (note: requires `eventLabel` to
+      // be a dimension in your report).
+      event_label: id
+
```

Comment:
Do you need to set the event's non-interaction flag when using GTM? I assume you still do?

Reply:
Yes you still do. But GTM sets this in the Tag Configuration with a dropdown. No need to send it explicit as a value.